.golangci.yml

```diff
@@ -30,6 +30,16 @@ run:
     - integration
 
 linters-settings:
+  depguard:
+    rules:
+      main:
+        # allow the following packages hierarchies
+        allow:
```

Comment:
I don't see any warnings when running `make lint`. But I guess if your IDE picks up on these, then it makes sense to bless them.